profile/README.md

```diff
@@ -1,27 +1,25 @@
+The OSCAL Compass project is a set of tools that enable the creation, validation, and governance of documentation artifacts for compliance needs. It leverages NIST's OSCAL ([Open Security Controls Assessment Language](https://pages.nist.gov/OSCAL/)) as a standard data format for interchange between tools and people, and provides an opinionated approach to OSCAL SDK and adoption by policy engines.
 
 The OSCAL Compass project is hosted by the [Cloud Native Computing Foundation (CNCF)](https://cncf.io)
 
-## Get Started
+## Getting Started
 
-Check out the [Community README](https://github.com/oscal-compass/community/blob/main/README.md) to get started with using and contributing to the project. The README also details all the ways to collaborate with project maintainers and your fellow users of OSCAL Compass tools. Anyone is welcome to participate and contribute provided they follow the OSCAL Compass [Code of Conduct](https://github.com/oscal-compass/community/blob/main/CODE_OF_CONDUCT.md).
+Check out the [Community page](https://oscal-compass.dev/community/) to get started with using and contributing to the project. This guide details all the ways to collaborate with project maintainers and your fellow users of OSCAL Compass tools. Anyone is welcome to participate and contribute provided they follow the OSCAL Compass [Code of Conduct](https://github.com/oscal-compass/community/blob/main/CODE_OF_CONDUCT.md).
 
-## Learn about the projects
+## Learn about the Projects
 
 [Trestle](https://github.com/oscal-compass/compliance-trestle) - Command line tool and SDK for interacting with OSCAL-based compliance-as-code documents
 
 [Agile Authoring](https://github.com/oscal-compass/compliance-trestle-agile-authoring) - Ready to use CI/CD pipeline configuration and setup using a GitOps approach and Trestle SDK for human and machine readable OSCAL compliance documents collaborative authoring. Manage semantic versioning, provenance traceability, change log, and approval based release to foster continuous compliance.
 
-[Compliance to Policy](https://github.com/oscal-compass/compliance-to-policy) (AKA C2P) - C2P is a plugin based tool to deploy compliance-as-code represented in OSCAL into policy validation or enforcement engines and collect and normalize their native results into OSCAL audit required format. Supported Policy Engines include [Kyverno](https://kyverno.io/) (for Kubernetes resources), [Open Cluster Management Policy Framework](https://open-cluster-management.io/) (for Kubernetes resources), [Auditree](https://auditree.github.io/) (generic).
+[Compliance to Policy](https://github.com/oscal-compass/compliance-to-policy) (AKA C2P) - C2P is a plugin-based toool that transforms compliance-as-code represented in OSCAL into native formats understood by policy validation or enforcement engines, and collects and normalizes their native results into OSCAL audit required format.
 
```

Comment:
typo - "toool" -> tool
Should we add artifacts - "deploy compliance-as-code **artifacts** represented in OSCAL".

Instead of removing the supported policy engines completely should we say something like - 
"It supports multiple Policy engines such as [Kyverno](https://kyverno.io/) (for Kubernetes resources), [Open Cluster Management Policy Framework](https://open-cluster-management.io/) (for Kubernetes resources), [Auditree](https://auditree.github.io/) (generic), and others".